libs/model-assessment/src/lib/ModelAssessmentDashboard/ModelAssessmentDashboardProps.ts

```diff
@@ -110,6 +111,14 @@ export interface IModelAssessmentDashboardProps
     abortSignal: AbortSignal
   ) => Promise<any>;
   requestExp?: (index: number, abortSignal: AbortSignal) => Promise<any[]>;
+  requestBubblePlotData?: (
```

Comment:
I feel like we have a lot of these request methods.  Any ideas on ways we could organize them better?

Reply:
Thanks Ilya for the comment. This is great idea! We can have a meeting with team to discuss about organizing these as we will have more requests methods to come in future.

Reply:
@imatiach-msft  we have an internal task in engineering excellence work to organize the new end points in some hierarchy.